lib/screens/leaderboards_screen.dart

```diff
@@ -21,7 +22,7 @@ class _Trophy extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return EmbeddedGameWidget(SimpleGame(SpriteComponent.fromSprite(size.x, size.y, _sprite)), size: size);
+    return EmbeddedGameWidget(BaseGame()..add(SpriteComponent.fromSprite(size.x, size.y, _sprite)), size: size);
```

Comment:
I tried using Flame.util.spriteAsComponent here but it doesn't seem to work... might be worth taking a look later.